frontend/src/modules/history/actions.js

```diff
@@ -223,11 +223,30 @@ export function deleteTranslation(
     }
 }
 
+export function deleteComment(
```

Comment:
Shouldn't this be in the `comments` module?

Reply:
The comments module does not have an `actions` or `reducer` it has been coming in through 'history', which is why this is here.